app/gui2/src/components/TableVizToolbar.vue

```diff
@@ -0,0 +1,247 @@
+}
+
+const filterPattern = computed(() => Pattern.parse('__ (__ __)'))
+
+const makeFilterPattern = (ast: Ast.Owned<Ast.MutableAst>, columnName: string, items: string[]) => {
+  if (
+    (items?.length === 1 && items.indexOf('true') != -1) ||
+    (items?.length === 1 && items.indexOf('false') != -1)
+  ) {
+    const boolToInclude = items.indexOf('false') != -1 ? Ast.parse('False') : Ast.parse('True')
+    return filterPattern.value.instantiateCopied([
+      Ast.TextLiteral.new(columnName),
+      Ast.parse('..Equal'),
+      boolToInclude,
+    ])
+  }
+  const itemList = items.map((i) => Ast.TextLiteral.new(i))
+  return filterPattern.value.instantiateCopied([
+    Ast.TextLiteral.new(columnName),
+    Ast.parse('..Is_In'),
+    Ast.Vector.new(ast.module, itemList),
+  ])
+}
+
+function getAstPatternSort() {
+  return Pattern.new((ast) =>
+    Ast.App.positional(
+      Ast.PropertyAccess.new(ast.module, ast, Ast.identifier('sort')!),
+      makeSortPattern(ast),
+    ),
+  )
+}
+
+function getAstPatternFilter(columnName: string, items: string[]) {
+  return Pattern.new((ast) =>
+    Ast.App.positional(
+      Ast.PropertyAccess.new(ast.module, ast, Ast.identifier('filter')!),
+      makeFilterPattern(ast, columnName, items),
+    ),
+  )
+}
+
+const createNewNodes = () => {
+  let patterns = new Array<any>()
+  if (Object.keys(props.filterModel).length) {
+    for (const index in Object.keys(props.filterModel)) {
+      const columnName = Object.keys(props.filterModel)[index]!
+      const items = props.filterModel[columnName || '']?.values.map((item) => `${item}`)!
+      const filterPatterns = getAstPatternFilter(columnName, items)
+      patterns.push(filterPatterns)
+    }
+  }
+  if (props.sortModel.length) {
+    const patSort = getAstPatternSort()
+    patterns.push(patSort)
+  }
+  config.createNodes(
+    ...patterns.map(
+      (pattern) => ({ content: pattern, commit: true }) satisfies NodeCreationOptions,
+    ),
+  )
+}
+
+const buttonClass = computed(() => {
+  return {
+    full: isFormatOptionSelected(TextFormatOptions.On),
+    partial: isFormatOptionSelected(TextFormatOptions.Partial),
+    strikethrough: isFormatOptionSelected(TextFormatOptions.Off),
+  }
+})
+
+const isFormatOptionSelected = (option: TextFormatOptions): boolean =>
+  option === textFormatterSelected.value
+
+const open = ref(false)
+const toggleOpen = () => {
+  open.value = !open.value
+}
+
+const changeFormat = (option: TextFormatOptions) => {
+  textFormatterSelected.value = option
+  toggleOpen()
+}
+</script>
+
+<template>
+  <div>
+    <DropdownMenu v-model:open="open" class="TextFormattingSelector" title="Text Display Options">
+      <template #button
+        ><div :class="buttonClass">
+          <SvgIcon name="paragraph" /></div
+      ></template>
+
+      <template #entries>
+        <MenuButton
+          class="full"
+          :title="`Text displayed in monospace font and all whitespace characters displayed as symbols`"
```

Comment:
```suggestion
          title="Text displayed in monospace font and all whitespace characters displayed as symbols"
```
And likewise for the other static-string titles below.